Automattic/jetpack
# Podcast Player: Increase podcast items

Fixes #21291

#### Changes proposed in this Pull Request:

* Updated podcast block extension react code to send `itemsToShow` to the WPcom API
* Updated WPcom API and its helper class to support `itemsToShow`

#### Jetpack product discussion NA



#### Does this pull request change what data or activity we track or use? NA



#### Testing instructions:






* Checkout to this PR branch and run project
* Create/edit a post
* Insert podcast block by typing `/podcast`
* Paste this URL `https://distributed.blog/category/podcast/feed/`  in the input field and submit it by clicking the right-hand side button
* Go to block settings
* open network tab using developer options
* Change **number of items** value `+` or `-`
* Inspect network calls and its response should show **tracks** property based on **number of items** 
* Changing **number of items** should also update the UI

**Screenrecording** https://youtu.be/H-2ttFYpdYI
